server/config/routes.js

```diff
@@ -93,6 +98,41 @@ exports.configure = function (app) {
   app.get('/location/province/:uuid', locations.lookupProvince);
   app.get('/location/detail/:uuid', locations.lookupDetail);
 
+  //API for account routes crud
+  app.get('/accounts/:list_type', account.list);
```

Comment:
":list_type" such be an optional parameter.  I would make it a query string.  The new route would be
```js
app.get('/accounts', accounts.list);

// inside accounts.list()
if (req.query.detailed) {
  // do some fancy SQL magic
}
// etc..
```